02-rendering-logic-ii/src/components/CharacterEditor/CharacterEditor.js

```diff
@@ -88,6 +40,55 @@ function App() {
           clothesColor={clothesColor}
         />
       </div>
+      <MaxWidthWrapper className={styles.maxWidthWrapper}>
+        <header className={styles.header}>
+          <h1 className={styles.title}>Create your Character</h1>
+          <p className={styles.description}>
+            Customize your character's look and style using the
+            controls below. What sort of adventure will you embark on?{' '}
+          </p>
+        </header>
+        <div className={styles.controlColumnWrapper}>
```

Comment:
Again: is this really necessary? I used this because it looks like it's the same problem described [here](https://courses.joshwcomeau.com/css-for-js/01-rendering-logic-1/07-margin#stretched-content)